content/en/docs/concepts/security/pod-security-standards.md

```diff
@@ -326,7 +326,7 @@ fail validation.
 		<tr>
 			<td style="white-space: nowrap">Privilege Escalation (v1.8+)</td>
 			<td>
-				<p>Privilege escalation (such as via set-user-ID or set-group-ID file mode) should not be allowed.</p>
+				<p>On Linux, do not allow privilege escalation (such as via set-user-ID or set-group-ID file mode). This control only applies when you set `spec.os` to <code>linux</code>, or if you do not specify a Pod OS. Versions of Kubernetes before v1.25 enforce this control regardless of operating system.</p>
```

Comment:
Rather than integrating this text in here, I'd suggest just having a standard header that you can add to all the linux-only fields. I prefer this approach because it makes it easier to identify the linux-only fields with a visual scan.

Maybe something like this:

```
<p><em><a href="#what-profiles-should-i-apply-to-my-windows-pods">Linux only</a> in v1.25+ (`spec.os != windows`)</em></p>
```

Reply:
It's a bit odd to hyperlink “Linux only” to `#what-profiles-should-i-apply-to-my-windows-pods` - maybe we could have a new page section that covers OS specifics, with a subheading for Linux and another subheading for Windows.